src/plugins/data/public/ui/typeahead/suggestion_component.tsx

```diff
@@ -61,6 +61,13 @@ interface Props {
 }
 
 export function SuggestionComponent(props: Props) {
+  // Removing empty suggestions from the history is for maintaining a clean user experience.
+  // Empty suggestions, which typically result from inadvertent keystrokes or incomplete queries,
+  // do not provide value to the user.
```

Comment:
I added a comment here, Would it be helpful to add [this link](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5348) to clarify the context of the original discussion?

Reply:
Yeah, in general links to GH issues or PRs with context is great. But the comment here is sufficient.